src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -846,3 +846,64 @@
+"""
+
+helps['spring-cloud build-service builder buildpack-binding'] = """
+    type: group
+    short-summary: (Support Enterprise Tier Only) Commands to manage buildpack-binding of builder.
+"""
+
+helps['spring-cloud build-service builder buildpack-binding create'] = """
+    type: command
+    short-summary: (Support Enterprise Tier Only) Create a buildpack binding.
+    examples:
+        - name: Create a buildpack binding without properties or secrets.
+          text: az spring-cloud build-service builder buildpack-binding create --name first-binding --builder-name first-builder --type ApplicationInsights --service MyCluster --resource-group MyResourceGroup
+        - name: Create a buildpack binding with only secrets.
+          text: az spring-cloud build-service builder buildpack-binding create --name first-binding --builder-name first-builder --type ApplicationInsights --secrets k1=v1 k2=v2 --service MyCluster --resource-group MyResourceGroup
+        - name: Create a buildpack binding with only properties.
+          text: az spring-cloud build-service builder buildpack-binding create --name first-binding --builder-name first-builder --type ApplicationInsights --properties a=b c=d --service MyCluster --resource-group MyResourceGroup
+        - name: Create a buildpack binding with properties and secrets.
+          text: az spring-cloud build-service builder buildpack-binding create --name first-binding --builder-name first-builder --type ApplicationInsights --properties a=b c=d --secrets k1=v1 k2=v2 --service MyCluster --resource-group MyResourceGroup
+"""
+
+helps['spring-cloud build-service builder buildpack-binding set'] = """
+    type: command
+    short-summary: (Support Enterprise Tier Only) Set a buildpack binding.
+    examples:
+        - name: Set a buildpack binding with properties and secrets.
+          text: az spring-cloud build-service builder buildpack-binding set --name first-binding --builder-name first-builder --type ApplicationInsights --properties a=b c=d --secrets k1=v1 k2=v2 --service MyCluster --resource-group MyResourceGroup
+"""
+
+helps['spring-cloud build-service builder buildpack-binding show'] = """
+    type: command
+    short-summary: (Support Enterprise Tier Only) Show a buildpack binding. The secrets will be masked.
+    examples:
+        - name: Show a buildpack binding.
+          text: az spring-cloud build-service builder buildpack-binding show --name first-binding --builder-name first-builder --service MyCluster --resource-group MyResourceGroup
+"""
+
+helps['spring-cloud build-service builder buildpack-binding list'] = """
+    type: command
+    short-summary: (Support Enterprise Tier Only) List all buildpack binding in a builder. The secrets will be masked.
+    examples:
+        - name: Show a buildpack binding.
+          text: az spring-cloud build-service builder buildpack-binding list --builder-name first-builder --service MyCluster --resource-group MyResourceGroup
+"""

```

Comment:
>  name: Show a buildpack binding.

Why is the description here `Show` rather than `list`?

Reply:
Good catch, thank you!

Reply:
Updated.